scripts/tests/test262/test262_whitelist.txt

```diff
@@ -104,26 +104,12 @@ language/expressions/object/method-definition/private-name-early-error-gen-insid
 language/expressions/object/method-definition/private-name-early-error-gen-inside-class.js(strict mode)
 language/expressions/object/method-definition/private-name-early-error-method-inside-class.js(default)
 language/expressions/object/method-definition/private-name-early-error-method-inside-class.js(strict mode)
-language/literals/bigint/non-octal-like-invalid-0008n.js(default)
-language/literals/bigint/non-octal-like-invalid-012348n.js(default)
-language/literals/bigint/non-octal-like-invalid-08n.js(default)
-language/literals/bigint/non-octal-like-invalid-09n.js(default)
-language/literals/bigint/numeric-separators/numeric-separator-literal-nonoctal-08-err.js(default)
-language/literals/bigint/numeric-separators/numeric-separator-literal-nonoctal-09-err.js(default)
-language/literals/bigint/numeric-separators/numeric-separator-literal-nonoctal-0_8-err.js(default)
-language/literals/bigint/numeric-separators/numeric-separator-literal-nonoctal-0_9-err.js(default)
-language/literals/bigint/numeric-separators/numeric-separator-literal-nzd-nsl-dds-leading-zero-err.js(default)
 language/literals/numeric/numeric-separators/numeric-separator-literal-lol-00-err.js(default)
 language/literals/numeric/numeric-separators/numeric-separator-literal-lol-01-err.js(default)
 language/literals/numeric/numeric-separators/numeric-separator-literal-lol-07-err.js(default)
 language/literals/numeric/numeric-separators/numeric-separator-literal-lol-0_0-err.js(default)
 language/literals/numeric/numeric-separators/numeric-separator-literal-lol-0_1-err.js(default)
 language/literals/numeric/numeric-separators/numeric-separator-literal-lol-0_7-err.js(default)
```

Comment:
If you want to open another PR to fix these tests, you are welcome to do so!

Reply:
Great, i would like to do so! Should i make an issue or just as a follow up of this PR?

Reply:
Naa, just a PR

Reply:
Goodd 👌